examples/typescript/aws-s3-static-website/cdktf.json

```diff
@@ -0,0 +1,12 @@
+{
+  "language": "typescript",
+  "app": "npx ts-node main.ts",
+  "projectId": "c00e7ea5-84b8-4a6a-80d5-985faf297684",
```

Comment:
We remove the `projectId` in examples to allow users to decide whether they want to opt into telemetry (if they copy the example and start from that)
```suggestion
```